README.md

```diff
@@ -88,8 +88,8 @@ This project has acceptance, integration and unit tests located inside the `test
 
 **Join the development**
 
-* Before you join development, please set up the project on your local machine, run it and go through the application completely. Press on any button you can find and see where it leads to. Explore. (Don't worry ... Nothing will happen to the app or to you due to the exploring :wink: Only thing that will happen is, you'll be more familiar with what is where and might even get some cool ideas on how to improve various aspects of the app.)
-* If you would like to work on an issue, drop in a comment at the issue. If it is already assigned to someone, but there is no sign of any work being done, please free to drop in a comment so that the issue can be assigned to you if the previous assignee has dropped it entirely.
+* Before you join development, please set up the project on your local machine, run it and go through the application completely. You can press on any button to find and see where it leads to. Explore. (Don't worry ... Nothing will happen to the app or to you due to the exploring :wink: Only thing that will happen is, you'll be more familiar with what is where and might even get some cool ideas on how to improve various aspects of the app.)
```

Comment:
There is a change in meaning here.

> Press on any button you can find and see where it leads to (what was already there)

Thus nudges the user to literally press on anything he/she can see on the website and see where it leads to. 

> You can press on any button to find and see where it leads to (Your change)

This tells the user that pressing on a button would help them see where it leads to. Which is pretty obvious. 

So, pleaser revert this change and let the previous phrase stay untouched.